### ISSUE TITLE
Re-designate hero images as alt text

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2764]

**Comment by Maroyafaied** [comment]
The alt text should be "decorative image"

**Comment by lpgoldstein** [comment]
@eelzi-ONRR Pleaser review that I've removed the descriptive alt text from the hero images and replaces with "" no hurry at all to check. 

https://dev-onrr-frontend.app.cloud.gov/?tabs=overview

- [ ] Home Hero image
- [ ] compliance and enforcement hero
- [ ] getting started hero
- [ ] Indian resources hero
- [ ] References hero
- [ ] About ONRR hero
- [ ] Paying hero
- [ ] Reporting hero

**Comment by eelzi-ONRR** [comment]
@lpgoldstein I checked the Home pages and a second page in each section and all the hero images alt text looks good! They all are ""

**Comment by lpgoldstein** [comment]
all live and checked.
